docs/src/examples/n-monitoring.md

```diff
@@ -189,8 +189,19 @@ The expected utility is used as the objective and the problem is solved using Gu
 ```julia
 EV = expected_value(model, diagram, x_s)
 @objective(model, Max, EV)
+```
 
+Alternatively, RJT formulation can be used by replacing commands on path compatibility variables and objective function creation with commands
 
+```julia
+μ_s = RJTVariables(model, diagram, z)
```

Comment:
Examples in the documentation should be updated to use the new generate_model function